<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WebDialogPlsLoadPlotPickAxis</class>
 <widget class="QWidget" name="WebDialogPlsLoadPlotPickAxis">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <property name="leftMargin">
    <number>32</number>
   </property>
   <property name="topMargin">
    <number>32</number>
   </property>
   <property name="rightMargin">
    <number>32</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="CaptionLabel">
       <property name="font">
        <font>
         <pointsize>24</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Pick other Axis</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="minimumSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../resource.qrc">
         <normaloff>:/resource/icons/icon_cross.png</normaloff>:/resource/icons/icon_cross.png</iconset>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="SelectGraphicViewLabel">
     <property name="text">
      <string>Select graphic view</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="Dimension2DViewRadioButton">
     <property name="text">
      <string>Plane view</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="Dim2XAxisLabel">
         <property name="text">
          <string>Horizontal axis (X)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="Dim2XAxisComboBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="Dim2YAxisLabel">
         <property name="text">
          <string>Vertical axis (Y)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="Dim2YAxisComboBox"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QRadioButton" name="SpectralViewRadioButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>SpectralView</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="VerticalAxisYLabel">
         <property name="text">
          <string>Vertical axis (Y)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="VerticalAxisYComboBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="GraphicTypeLable">
         <property name="text">
          <string>Graphic</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="TabSwitcher" name="GraphicStyleSwitcher" native="true"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Maximum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TabSwitcher</class>
   <extends>QWidget</extends>
   <header>ui/controls/tab_switcher.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resource.qrc"/>
 </resources>
 <connections/>
</ui>
